ngraph/test/op_eval/reduce_min.cpp

```diff
@@ -0,0 +1,258 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
Can you switch from `Backend` to `TestCase` for all the tests? We should be going away from explicit backend creation because the `TestCase` class allows parametrization with engines.

Reply:
I would say that for "op_eval" tests, even more appropriate is using explicit `ngraph::Function::evaluate()` than any of "backend" with `TestCase` or not.
So `test/backend` for TestCase with backends, `test/op_eval` (if needed) for explicit evaluate call.

Reply:
I agree that we should switch to `TestCase`, but there are separate tasks created for this in the past, including reduce ops, eg 35880, 35197. Refactor backend tests to TestCase is not part of requirements of ref impl. What we are doing is that every new test introduced uses TestCase. For refactoring, as it involves significant amount of work, we will assign those already created tasks and allocate them in the plan